src/Components/CustomElements/src/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,3 @@
+﻿#nullable enable
+Microsoft.AspNetCore.Components.Web.CustomElementsJSComponentConfigurationExtensions
+static Microsoft.AspNetCore.Components.Web.CustomElementsJSComponentConfigurationExtensions.RegisterAsCustomElement<TComponent>(this Microsoft.AspNetCore.Components.Web.IJSComponentConfiguration! configuration, string! customElementName) -> void
```

Comment:
Are you planning on following the API review process?
https://github.com/dotnet/aspnetcore/blob/main/docs/APIReviewProcess.md

Reply:
https://github.com/dotnet/aspnetcore/issues/42329